src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeClientTests.cs

```diff
@@ -207,6 +207,78 @@ public async Task TSConfigFiles_HasResponse_DeserializedResponse()
             result.Should().BeEquivalentTo(response);
         }
 
+        [TestMethod]
```

Comment:
I'm guessing these test are copies of the JS ones with js->ts changes?

Reply:
Yep; sorry I should've said that.